src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -574,10 +574,16 @@ private function registerAssetsConfiguration(array $config, ContainerBuilder $co
     {
         $loader->load('assets.xml');
 
+        $removeGitStrategy = true;
```

Comment:
This logic is for developers who not use this strategy at all.

They can have git not installed, or not properly php configured to exec Process functionality and so on. They will be very disappointed if GitHash compiler will try to get git hash for them.

And this logic just say to GitHashCompilerPass - "hey, developer do not use this functionality, just skip it". 

If I will remove this logic - how GitHashCompilerPass can understand that this hash is not needed to be processed?